interfaces/Portalicious/src/app/pages/project-registrations/components/change-status-dialog/change-status-dialog.component.html

```diff
@@ -0,0 +1,303 @@
+        "Completed" status. Registrations with “completed” status will remain
+        with their current status.
+      </p>
+    }
+  }
+  <p
+    i18n
+    class="pt-2"
+  >
+    Would you like to proceed with
+    {{ changeStatusMutation.data()?.applicableCount }}
+    registration(s)?
+  </p>
+</app-confirmation-dialog>
+
+<p-dialog
+  i18n-header
+  [(visible)]="dryRunFailure"
+  [modal]="true"
+  [dismissableMask]="true"
+>
+  <ng-template pTemplate="header">
+    <h3>
+      <i class="pi pi-exclamation-triangle me-2"></i>
+      <ng-container i18n>Error</ng-container>
+    </h3>
+  </ng-template>
+  <b i18n>0 selected registrations support this action. </b>
+  @switch (status()) {
+    @case (RegistrationStatusEnum.validated) {
+      <p
+        i18n
+        class="pt-2"
+      >
+        The action "Validate" can only be applied to registrations with the
+        "Registered" status.
+      </p>
+      <p
+        i18n
+        class="pt-2"
+      >
+        Select registrations with the status “Registered” and try again.
+      </p>
+    }
+    @case (RegistrationStatusEnum.included) {
+      <p
+        i18n
+        class="pt-2"
+      >
+        The action "Include" can only be applied to registrations who's
+        “Payments left” is larger than 0.
+      </p>
+      <p
+        i18n
+        class="pt-2"
+      >
+        Select registrations who's “Payments left” is larger than 0 and try
+        again.
+      </p>
+    }
+    @case (RegistrationStatusEnum.paused) {
+      <p
+        i18n
+        class="pt-2"
+      >
+        The action "Pause" can only be applied to registrations with the
+        "Included" status.
+      </p>
+      <p
+        i18n
+        class="pt-2"
+      >
+        Select registrations with the status “included” and try again.
+      </p>
+    }
+    @case (RegistrationStatusEnum.declined) {
+      <p
+        i18n
+        class="pt-2"
+      >
+        The action "Decline" can not be applied to registrations with the
+        "Completed" status.
+      </p>
+      <p
+        i18n
+        class="pt-2"
+      >
+        Select registrations with other status and try again.
+      </p>
+    }
+    @case (RegistrationStatusEnum.deleted) {
+      <p
+        i18n
+        class="pt-2"
+      >
+        The action "Delete" can not be applied to registrations with the
+        "Completed" status.
+      </p>
+      <p
+        i18n
+        class="pt-2"
+      >
+        Select registrations with other status and try again.
+      </p>
+    }
+    @default {
+      <p
+        i18n
+        class="pt-2"
+      >
+        This action can not be applied to the registrations you have selected.
+      </p>
+      <p
+        i18n
+        class="pt-2"
+      >
+        Select registrations with another status and try again.
+      </p>
+    }
+  }
+  <div class="mt-6 flex justify-end space-x-2">
+    <p-button
+      label="OK"
+      i18n-label="@@generic-ok"
```

Comment:
```suggestion
      label="Cancel"
      i18n-label="@@generic-cancel"
```

Better IMHO but I don't feel strongly about it